sensor/common/detector/baseline/baseline_test.go

```diff
@@ -9,7 +12,174 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var (
+	benchMax = flag.Bool("bench.max", false, "Run maximum scale benchmarks (300k containers)")
+)
+
+// benchmarkMemoryUsage is a parameterized benchmark function
+func benchmarkMemoryUsage(b *testing.B, evaluatorFactory func() Evaluator, baselines []*storage.ProcessBaseline, scenarioName string, showDeduplication bool) {
```

Comment:
I love benchmarks! Could we maybe place this one in a `baseline_bech_test.go` file? I am not sure whether we already follow this pattern, but separating benchmarks from tests may be desirable.